var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -211,7 +211,7 @@ class NetcdfC(CMakePackage, AutotoolsPackage):
         # The macro usage was adjusted (required when versions 1.8.23+, 1.10.8+, 1.12.1+ and
         # 1.13.0+ of HDF5 are used) in NetCDF 4.8.1
         # (see https://github.com/Unidata/netcdf-c/pull/2034):
-        depends_on("hdf5@:1.8.22,1.10.0:1.10.7,1.12.0,1.13:", when="@:4.8.0")
```

Comment:
Although it doesn't change much due to other constraints, I'd prefer to see `,1.14:` in this context because the directive documents a particular `byterange`-related issue. Also, it will be still possible to install `hdf5@1.13:1.13` even without the checksums. So, I don't think this change is needed at all. Could you, please, revert it? Thanks.

Reply:
Done.  Thank you!